### ISSUE TITLE
Ensure that the step width must be positive

### ISSUE THREAD

**Issue by florianpircher** [tcarisland/GradientHatch#40]
Currently, one can set a step width of 0 or a negative number and then Glyphs is stuck in an infinite loop.

**Comment by tcarisland** [comment]
I noticed that when I reproduced the bug, it also causes the Glyphs app to crash any other time I try running the plugin again.

This is because of how Glyphs caches the values used in the filters.

To fix this. comment out the call to the runFilter method in GradientHatch.glyphsFilter/Resources-directory (currently line 212).

It should then be possible to add a positive value and then rerun the plugin when you've managed to input a positive value again.

`self.runFilter(layer, angle, offsetPath, [offsetPathStart, offsetPathEnd], stepWidth, [originX, originY], expandBeforeInset)`

![Skjermbilde 2024-02-14 kl  22 17 44](https://github.com/tcarisland/GradientHatch/assets/11506194/ab6094b5-a79d-4cda-9485-d95be51618bf)
